test/recipes/01-test_symbol_presence.t

```diff
@@ -9,15 +9,22 @@
 
 use strict;
 use File::Spec::Functions qw(devnull);
-use OpenSSL::Test qw(:DEFAULT srctop_file bldtop_dir bldtop_file);
+use OpenSSL::Test qw(:DEFAULT srctop_dir srctop_file bldtop_dir bldtop_file);
 use OpenSSL::Test::Utils;
 
-setup("test_symbol_presence");
+#Load configdata.pm
+
+BEGIN {
+    setup("test_symbol_presence");
+}
+use lib srctop_dir('Configurations');
+use lib bldtop_dir('.');
+use platform;
 
 plan skip_all => "Only useful when building shared libraries"
     if disabled("shared");
 
-my @libnames = ("crypto", "ssl");
+my @libnames = ("crypto", "ssl", "fips");
```

Comment:
This list could really be refactored to:

``` perl
my @libnames = ("libcrypto", "libssl", "fips");
```

That way, you don't have to add "lib" all over the place.

Also, it might be prudent to make this an associative list, to map libraries/modules against their corresponding ordinals file, so maybe something like this?

``` C
my %libnames = (
    libcrypto => 'libcrypto',
    libssl    => 'libssl',
    fips      => 'provider'
);
```

And then rework the code to use that table to figure out the correct ordinals file to compare with.